From 98dfbb7eafe7a370cdfb8b93e301ca7795f25767 Mon Sep 17 00:00:00 2001
From: "Robert T. McGibbon" <rmcgibbo@gmail.com>
Date: Sat, 2 Jan 2016 22:12:50 -0800
Subject: [PATCH 1/1] Use more portable expression in configure

---
 configure | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git configure configure
index 29ecbd2..77d60aa 100755
--- configure
+++ configure
@@ -397,7 +397,7 @@ VALUE=""
 while [[ ! -z $1 ]] ; do
   # If arg is Key=Value, separate into Key and Value
   VALUE=""
-  POS=`expr index "$1" '='`
+  POS=`echo $1 | awk 'match($0,"="){print RSTART}'`
   if [[ $POS -eq 1 ]] ; then
     echo "Error: '=' cannot be first character in an argument." > /dev/stderr ; exit 1
   elif [[ $POS -gt 1 ]] ; then
--
2.3.4

