From d203ba29719927c0df34faf59f8dab823f836f4c Mon Sep 17 00:00:00 2001
From: Robert McGibbon <rmcgibbo@gmail.com>
Date: Sun, 18 Oct 2015 22:46:04 -0700
Subject: [PATCH] Force zip_safe=False

---
 setup.py | 1 +
 1 file changed, 1 insertion(+)

diff --git setup.py setup.py
index 031ad53..b346c7d 100644
--- setup.py
+++ setup.py
@@ -246,6 +246,7 @@ language.''',
     ext_modules = extmods,
     package_dir = {"cvxopt": "src/python"},
     packages = ["cvxopt"],
+    zip_safe=False,
     classifiers=[
         'Development Status :: 5 - Production/Stable',
         'Intended Audience :: Science/Research',
-- 
2.3.4

