From 04b17f43adca97c770085352e949d283bca64850 Mon Sep 17 00:00:00 2001
From: "Robert T. McGibbon" <rmcgibbo@gmail.com>
Date: Thu, 10 Mar 2016 00:25:13 -0500
Subject: [PATCH 1/1] Fix 'rb' for Python 3

---
 GPy/util/datasets.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git GPy/util/datasets.py GPy/util/datasets.py
index f917336..45e204e 100644
--- GPy/util/datasets.py
+++ GPy/util/datasets.py
@@ -54,12 +54,12 @@ on_rtd = os.environ.get('READTHEDOCS', None) == 'True' #Checks if RTD is scannin

 if not (on_rtd):
     path = os.path.join(os.path.dirname(__file__), 'data_resources.json')
-    json_data=open(path).read()
+    json_data=open(path, 'rb').read().decode('utf-8')
     data_resources = json.loads(json_data)

 if not (on_rtd):
     path = os.path.join(os.path.dirname(__file__), 'football_teams.json')
-    json_data=open(path).read()
+    json_data=open(path, 'rb').read().decode('utf-8')
     football_dict = json.loads(json_data)


--
2.6.4

