*** setup.py	2019-01-01 19:43:19.000000000 -0800
--- setup.py	2019-01-01 19:43:38.000000000 -0800
***************
*** 54,59 ****
      packages=find_packages(),
      package_data={'openmmsetup': find_package_data()},
      zip_safe=False,
-     install_requires=['flask', 'openmm >= 7.2', 'pdbfixer >= 1.5'],
      entry_points={'console_scripts': ['openmm-setup = openmmsetup.openmmsetup:main']})

--- 54,58 ----
