--- setup.py.back	2015-04-22 17:06:04.725990857 -0700
+++ setup.py		2015-04-22 17:04:20.313992683 -0700
@@ -58,7 +58,7 @@ def main():
             library_dirs=LIBRARY_DIRS + EXTRA_LIBRARY_DIRS,
             libraries=LIBRARIES + EXTRA_LIBRARIES,
             extra_compile_args=conf["CXXFLAGS"],
-            define_macros=list(EXTRA_DEFINES.iteritems()),
+            define_macros=list(EXTRA_DEFINES.items()),
             ))

         requirements.append("PyUblas>=0.92.1")
@@ -117,16 +117,16 @@ def main():
                'Topic :: Software Development :: Libraries',
                ],

-            author=u"Andreas Kloeckner",
+            author="Andreas Kloeckner",
             author_email="inform@tiker.net",
             license="MIT",
             url="http://mathema.tician.de/software/pyvisfile",

             # dependencies
-            setup_requires=requirements,
-            install_requires=[
-                    "pytools>=2013.2"
-                    ] + requirements,
+            # setup_requires=requirements,
+            # install_requires=[
+            #         "pytools>=2013.2"
+            #         ] + requirements,

             packages=[
                     "pyvisfile",