From 4ccffb790d4adfbd8c119855b083f1301a5264b9 Mon Sep 17 00:00:00 2001
From: Matthew Harrigan <harrigan@stanford.edu>
Date: Wed, 9 Nov 2016 17:00:19 -0800
Subject: [PATCH] Install path for zlib on linux (too)

---
 setup.py | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index e3ac9e6..02be8a9 100644
--- a/setup.py
+++ b/setup.py
@@ -119,10 +119,14 @@ def format_extensions():
     if sys.platform == 'win32':
         # Conda puts the zlib headers in ./Library/... on windows
         # If you're not using conda, good luck!
-        # (on linux, zlib is a dependency of python and its headers/libraries
-        #  go in the normal ./include ./lib directories)
         zlib_include_dirs += ["{}/Library/include".format(sys.prefix)]
         zlib_library_dirs += ["{}/Library/lib".format(sys.prefix)]
+    else:
+        # On linux (and mac(?)) these paths should work for a standard
+        # install of python+zlib or a conda install of python+zlib
+        zlib_include_dirs += ["{}/include".format(sys.prefix)]
+        zlib_library_dirs += ["{}/lib".format(sys.prefix)]
+
     tng = Extension('mdtraj.formats.tng',
                     sources=glob('mdtraj/formats/tng/src/compression/*.c') +
                                 ['mdtraj/formats/tng/src/lib/tng_io.c',
