diff --git a/mdtraj/formats/tng/src/lib/tng_io.c b/mdtraj/formats/tng/src/lib/tng_io.c
index e4a70542..5ef7a7c6 100644
--- a/mdtraj/formats/tng/src/lib/tng_io.c
+++ b/mdtraj/formats/tng/src/lib/tng_io.c
@@ -4692,10 +4692,11 @@ static tng_function_status tng_gzip_uncompress(const tng_trajectory_t tng_data,
                                                const int64_t uncompressed_len)
 {
     Bytef *dest;
+    uLongf new_len;
     unsigned long stat;
     (void)tng_data;
-    uLongf new_len = uncompressed_len;
 
+    new_len = uncompressed_len;
     dest = malloc(uncompressed_len);
     if(!dest)
     {
