diff --git a/mdtraj/geometry/include/math_patch.h b/mdtraj/geometry/include/math_patch.h
index 742770f5..2db5c091 100644
--- a/mdtraj/geometry/include/math_patch.h
+++ b/mdtraj/geometry/include/math_patch.h
@@ -1,7 +1,7 @@
 // Math functions for old versions of visual studio
 
 #ifdef _MSC_VER
-#if _MSC_VER < 1900 // prior to vs2015
+//#if _MSC_VER < 1900 // prior to vs2015
 float roundf(float x) {
    return x >= 0.0f ? floorf(x + 0.5f) : ceilf(x - 0.5f);
 }
@@ -11,7 +11,7 @@ double cbrt(double x) {
     else
         return pow(x, 1.0/3.0);
 }
-#endif // _MSC_VER < 1900
+//#endif // _MSC_VER < 1900
 
 #ifndef isnan
 #define isnan(x) ((x)!=(x))
