diff --git a/mdtraj/formats/tng/src/lib/tng_io.c b/mdtraj/formats/tng/src/lib/tng_io.c
index e4a70542..4633fd3c 100644
--- a/mdtraj/formats/tng/src/lib/tng_io.c
+++ b/mdtraj/formats/tng/src/lib/tng_io.c
@@ -4693,9 +4693,9 @@ static tng_function_status tng_gzip_uncompress(const tng_trajectory_t tng_data,
 {
     Bytef *dest;
     unsigned long stat;
+    uLongf new_len;
     (void)tng_data;
-    uLongf new_len = uncompressed_len;
-
+    new_len = uncompressed_len;
     dest = malloc(uncompressed_len);
     if(!dest)
     {
